client/lib/hoverIntent/index.js

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
It's a bit of a bummer to use this library without any credits or mention of the original author. Thoughts?

Reply:
> This project is licensed under the MIT License

https://opensource.org/licenses/MIT

Not sure how relevant here but the license text includes:

> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Reply:
I think we usually add a mention to the library and the license in `CREDITS.md`

Reply:
Yep, a mention in `CREDITS.md` should do.